README.md

```diff
@@ -128,11 +133,11 @@ OPTIONS
 
 EXAMPLE
     
-    Collect data about jobs started a week ago and earlier and put them to the
-    InfluxDB:
+    Collect data about jobs and tests started a week ago and earlier and put
+    them to the InfluxDB:
```

Comment:
'earlier' means a week ago or before that.

data is singular, so them→'it' or 'this data'

```suggestion
    Collect data about jobs and tests which started a week ago or later,
    then put this data to InfluxDB:
```

Reply:
done